x/staking/simulation/operations.go

```diff
@@ -41,15 +39,17 @@ func WeightedOperations(
 	bk types.BankKeeper, k keeper.Keeper,
 ) simulation.WeightedOperations {
 	var (
-		weightMsgCreateValidator             int
-		weightMsgEditValidator               int
-		weightMsgDelegate                    int
-		weightMsgUndelegate                  int
-		weightMsgBeginRedelegate             int
-		weightMsgCancelUnbondingDelegation   int
-		weightMsgTokenizeShares              int
-		weightMsgRedeemTokensforShares       int
-		weightMsgTransferTokenizeShareRecord int
+		weightMsgCreateValidator int
+		weightMsgEditValidator   int
+		weightMsgDelegate        int
+		weightMsgUndelegate      int
+		weightMsgBeginRedelegate int
+		// weightMsgCancelUnbondingDelegation int
```

Comment:
Should we remove these if unused?

Reply:
we should fix these, but lets do it in a separate PR